src/drawers/drawer.ts

```diff
@@ -111,16 +111,22 @@ export module Drawers {
      *
      * @returns {d3.Selection} the renderArea selection
      */
-    public _getRenderArea() {
+    public renderArea() {
       return this._renderArea;
     }
 
-    public _getSelector(): string {
+    /**
+     * Returns the selector for this Drawer's visual elements.
```

Comment:
What is a selector? I think the comment should be more precise, or the method should be renamed